protocol-definitions/Metrics.yaml

```diff
@@ -0,0 +1,40 @@
+id: 39
+name: Metrics
+methods:
+  - id: 1
+    name: readMetrics
+    since: 2.0
+    doc: |
+      Reads the recorded metrics starting with the smallest sequence number
+      greater or equals to the sequence number set in fromSequence.
+    request:
+      retryable: false
```

Comment:
the call seems to be idempotent since it is a read-only method, hence, can it be retryable?

Reply:
@blazember is this an idempotent call?

Reply:
Yes, it can be retryable.